lib/ui/window/platform_configuration_unittests.cc

```diff
@@ -271,5 +271,40 @@ TEST_F(ShellTest, PlatformConfigurationOnErrorThrows) {
   DestroyShell(std::move(shell), std::move(task_runners));
 }
 
+TEST_F(ShellTest, PlatformConfigurationSetDartPerformanceMode) {
+  auto message_latch = std::make_shared<fml::AutoResetWaitableEvent>();
+  auto finish = [message_latch](Dart_NativeArguments args) {
+    // call needs to happen on the UI thread.
+    Dart_PerformanceMode prev =
+        Dart_SetPerformanceMode(Dart_PerformanceMode_Default);
+    ASSERT_EQ(Dart_PerformanceMode_Latency, prev);
```

Comment:
@rmacnak-google this assert seems to be consistently failing returning an invalid performance mode. Looks like the value returned seems to be invalid in a few cases, could you please take a look?